proof/infoflow/ARM/Example_Valid_State.thy

```diff
@@ -1,5 +1,6 @@
 (*
  * Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+ * Copyright 2023, Proofcraft Pty Ltd
```

Comment:
Usually I put the most recent ones at the top so that we get reverse chronological order. I'm not too hung up on the particular order, but probably better to do it consistently.